layouts/404.html

```diff
@@ -9,9 +9,9 @@
         <div class="grid-container">
           <div class="grid-row grid-gap">
             <div class="tablet:grid-col-4">
-              <a class="site-logo" href="{{ .Site.BaseURL }}" accesskey="1" title="Home" aria-label="Home">
+              <a class="site-logo" href="https://vote.gov/" accesskey="1" title="Home" aria-label="Home">
                 <div class="logo-text">Vote.gov</div>
-                <img id="SiteLogo" src="{{ .Site.BaseURL }}/assets/img/ballot_box-blue.svg" alt="Web design standards">
+                <img id="SiteLogo" src="https://vote.gov/assets/img/ballot_box-blue.svg" alt="Web design standards">
```

Comment:
Let's remove the changes to the 404.html from this PR. This task is specifically only for metatags.